run/pubsub/main.py

```diff
@@ -41,8 +45,10 @@ def index():
     if isinstance(pubsub_message, dict) and 'data' in pubsub_message:
         name = base64.b64decode(pubsub_message['data']).decode('utf-8').strip()
 
-    print(f'Hello {name}!')
-    return ('', 204)
+    msg = f'Hello {name}!'
+    logging.info(msg)
+
+    return msg
```

Comment:
This change diverges the sample from the Node.js and Go samples, which return empty/204 responses. Returning the msg as part of the HTTP response is not used by Pub/Sub. I think this change is fine but we should have a comment indicating it is for testing.